docs/experiments-monot5-gpu.md

```diff
@@ -11,30 +11,20 @@ We rerank the BM25 run files that contain ~1000 passages per query using monoT5.
 monoT5 is a pointwise reranker. This means that each document is scored independently using T5.
 
 ## Environment Setup
-This experiment is proved to work on Tensorflow-2.3.0. Please make sure you have required gcc version accourding to [TensorFlow gpu dependency](https://www.tensorflow.org/install/source#gpu). 
-
-
-```
-module list             # Check if you have coreect version of gcc
-module spider gcc       # Check available gcc
-module load gcc/7.3.0   # Load required gcc
-```
-
 Creat a Python virtual environment for the experiments and install the dependncies
 
 If you haven't installed Anaconda on Compute Canada, please follow this guide [here](https://www.digitalocean.com/community/tutorials/how-to-install-anaconda-on-ubuntu-18-04-quickstart)
 
-Within Anaconda virtual environment, please install packges with `conda install ` first rather than `pip install` to avoid dependcy conflict
 ```
 conda init
 conda create --y --name pygaggle python=3.6
 conda activate pygaggle
 unset PYTHONPATH
 conda install -c conda-forge httptools jsonnet --yes
-conda install -c tensorflow-gpu==2.3.0
+pip install tensorflow-gpu==2.3.0
```

Comment:
why we cannot use conda for these 2 packages?

Reply:
We can use conda to install it. 
theoretically, both conda and pip should be ok. I encounter the issue that tensorflow cant find the package which is resolved by using conda install. But I don't think this would happen to everyone. So I just change it back and mention that problem in trouble shooting sections